testsuite/tests/backtrace/backtrace_effects.byte.reference

```diff
@@ -0,0 +1,12 @@
+(** get_callstack **)
+Raised by primitive operation at Backtrace_effects.bar in file "backtrace_effects.ml", line 20, characters 13-39
+Called from Backtrace_effects.foo in file "backtrace_effects.ml", line 27, characters 12-17
+Called from Backtrace_effects in file "backtrace_effects.ml", line 43, characters 8-14
+(** get_continuation_callstack **)
+Raised by primitive operation at Backtrace_effects.bar in file "backtrace_effects.ml", line 22, characters 4-13
+Called from Backtrace_effects.foo in file "backtrace_effects.ml", line 27, characters 12-17
+(** raise **)
+Fatal error: exception Stdlib.Exit
+Raised at Backtrace_effects.bar in file "backtrace_effects.ml", line 17, characters 4-14
+Re-raised at Backtrace_effects.baz.(fun) in file "backtrace_effects.ml", line 33, characters 21-28
+Called from Backtrace_effects in file "backtrace_effects.ml", line 43, characters 8-14
```

Comment:
Is this file used? It doesn't seem referenced from `backtrace_effects.ml`

Reply:
Oops, edited the wrong test at some point. Fixed.